src/styles/normalize.css

```diff
@@ -0,0 +1,24 @@
+* {
```

Comment:
why did you create two files, normalise/normalize?

Reply:
forgot to delete one of them, fixed